src/libraries/System.Security.Cryptography.Algorithms/tests/Rfc2898Tests.cs

```diff
@@ -346,7 +329,6 @@ public static void GetBytes_KnownValues_4()
                 });
         }
 
-#if NETCOREAPP
```

Comment:
Unrelated test cleanup: these tests don't build for anything other the `NETCOREAPP`, so the `#if` is not needed.